pgcli/main.py

```diff
@@ -366,9 +372,17 @@ def run_cli(self):
 
         except EOFError:
             print ('Goodbye!')
-        finally:  # Reset the less opts back to original.
-            logger.debug('Restoring env var LESS to %r.', original_less_opts)
-            os.environ['LESS'] = original_less_opts
+        finally:
+            # Don't persist pgcli less defaults when session is terminated
+            if less_opts_adjusted:
+                os.environ['LESS'] = ''
+
+    def display_output(self, output, output_fits_screen=True):
+        if self.pgspecial.pager_config == special.main.PAGER_ALWAYS or \
+            (self.pgspecial.pager_config == special.main.PAGER_LONG_OUTPUT and not output_fits_screen):
+            click.echo_via_pager(output)
+        else:
+            print(output)
```

Comment:
I'd prefer to use `click.echo(output)` since it tries to be a robust replacement for print(). http://click.pocoo.org/5/quickstart/#echoing